open-metadata-implementation/admin-services/admin-services-registration/src/main/java/org/odpi/openmetadata/adminservices/configuration/OMAGAccessServiceRegistration.java

```diff
@@ -2,7 +2,7 @@
 /* Copyright Contributors to the ODPi Egeria project. */
 package org.odpi.openmetadata.adminservices.configuration;
 
-import org.odpi.openmetadata.adminservices.configuration.registration.AccessServiceRegistration;
+import org.odpi.openmetadata.adminservices.configuration.registration.ServiceRegistration;
```

Comment:
Is the plan to eventual merge these different `OMAG...ServiceRegistration` classes into one?  They're looking pretty identical now that the `ServiceRegistration` class they register is identical across all of them (?)

Reply:
Raised issue #2593 to track this.